sql/core/src/main/scala/org/apache/spark/sql/execution/window/AggregateProcessor.scala

```diff
@@ -113,7 +113,7 @@ private[window] object AggregateProcessor {
  * This class manages the processing of a number of aggregate functions. See the documentation of
  * the object for more information.
  */
-private[window] final class AggregateProcessor(
+private[sql] final class AggregateProcessor(
```

Comment:
Do you mean we can remove the modifier here altogether because the fact that the class lives under namespace "execution" implies it's private already?

Reply:
Yea, see also some JIRAs like SPARK-16964.

Reply:
Thanks. I have removed the private[package].